.travis.yml

```diff
@@ -59,22 +59,6 @@ jobs:
     - make e2e-test
   # split builds to avoid job timeouts
   - stage: publish amd64
-    if: type = api AND branch = master AND repo = kubernetes/ingress-nginx
+    if: type = push AND branch = master AND repo = Shopify/ingress
     script:
     - travis_wait ${TRAVIS_TIMEOUT} .travis/publish.sh amd64
-  - stage: publish arm
-    if: type = api AND branch = master AND repo = kubernetes/ingress-nginx
-    script:
-    - travis_wait ${TRAVIS_TIMEOUT} .travis/publish.sh arm
-  - stage: publish arm64
-    if: type = api AND branch = master AND repo = kubernetes/ingress-nginx
-    script:
-    - travis_wait ${TRAVIS_TIMEOUT} .travis/publish.sh arm64
-  - stage: publish ppc64le
-    if: type = api AND branch = master AND repo = kubernetes/ingress-nginx
-    script:
-    - travis_wait ${TRAVIS_TIMEOUT} .travis/publish.sh ppc64le
-  - stage: publish s390x
-    if: type = api AND branch = master AND repo = kubernetes/ingress-nginx
-    script:
-    - travis_wait ${TRAVIS_TIMEOUT} .travis/publish.sh s390x
```

Comment:
this does not seem like a necessary change to do, I'd leave them around

Reply:
The reason why I removed them is that the CI build time is > 50 minutes if we build and publish the images for all the architectures. This is a problem because travis CI will timeout and stop the build if any job takes more than 50 minutes (https://docs.travis-ci.com/user/customizing-the-build#Build-Timeouts). What are your thoughts? @ElvinEfendi

Reply:
but if you keep the conditions as they are they won't be triggered, will they?

Reply:
You are right :)